src/App.tsx

```diff
@@ -1,9 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { SearchField } from './components/SearchField/SearchField';
+
+function getPreparedMovies(movies: Movie[], query:string) {
+  let preparedMovies = [...movies];
+
+  if (query) {
+    const searchPropmpt = query.toLowerCase().trim();
+
+    preparedMovies = preparedMovies.filter(
+      movie => movie.title.toLowerCase().includes(searchPropmpt)
+    || movie.description.toLowerCase().includes(searchPropmpt),
+    );
+  }
+
+  return preparedMovies;
```

Comment:
let's improve your solution 
```suggestion
 if (!query) {
  return movies;
 }
 const searchPropmpt = query.toLowerCase().trim();
 return preparedMovies.filter(
   movie =>
     movie.title.toLowerCase().includes(searchPropmpt) ||
     movie.description.toLowerCase().includes(searchPropmpt)
 );
```